internal/pkg/externalplugins/labelblocker/labelblocker.go

```diff
@@ -105,6 +106,27 @@ func HandlePullRequestEvent(gc githubClient, pullRequestEvent *github.PullReques
 	return handle(cfg, ctx, gc, log)
 }
 
+// HandleIssueEvent handles a GitHub issue event.
+func HandleIssueEvent(gc githubClient, issueEvent *github.IssueEvent,
+	cfg *externalplugins.Configuration, log *logrus.Entry) error {
+	// Only consider the labeled / unlabeled actions.
+	if issueEvent.Action != github.IssueActionLabeled &&
```

Comment:
Should we only reps to opening issue or PR?

Reply:
I don’t think it should be. If it is a general plugin, it should not limit the status of the issue or PR.

Reply:
OK